main.js

```diff
@@ -0,0 +1,155 @@
+// I have not yet finished my AI function \\
+// My counter does not reset the minutes after each hour, cant figure out why \\
+// If invalid move clicked game ends, need to figure that out... \\
+
+
+//-----------------------------------------------------------------------------------------------------------------------------
+// Variables to be used throughout code \\
+
+let player1 = document.getElementById('1player');
+let player2 = document.getElementById('2player');
+let status = document.getElementById('status');
+let reset = document.getElementById('reset');
+
+let gameBoard = document.getElementsByClassName('board')
+let row1 = document.getElementById('row1')
+let cell1 = document.getElementById('cell1')
+let cell2 = document.getElementById('cell2')
+let cell3 = document.getElementById('cell3')
+
+let row2 = document.getElementById('row2')
+let cell4 = document.getElementById('cell4')
+let cell5 = document.getElementById('cell5')
+let cell6 = document.getElementById('cell6')
+
+let row3 = document.getElementById('row3')
+let cell7 = document.getElementById('cell7')
+let cell8 = document.getElementById('cell8')
+let cell9 = document.getElementById('cell9')
+
+let aiPlayer = "O";
+let player = "X";
+let Winner = null;
+
+let board = [
+    cell1, cell2, cell3,
+    cell4, cell5, cell6,
+    cell7, cell8, cell9
+]
+
+let prefferedMove = {
+    move1: cell1,
+    move2: cell2,
+    move3: cell3,
+    move4: cell4,
+    move5: cell5,
+    move6: cell6,
+    move7: cell7,
+    move8: cell8,
+    move9: cell9,
+}
+
+//-----------------------------------------------------------------------------------------------------------------------------
+
+function startTimer() {
+    let seconds = 0;
+    timer = setInterval(function () {
+        seconds++;
+        document.getElementById("seconds").innerText = seconds % 60;
+        document.getElementById("minutes").innerText = parseInt(seconds / 60);
+        document.getElementById("hours").innerText = parseInt(seconds / 3600);
+    }, 1000);
+}
+
+function stopTimer() {
+    clearInterval(timer);
+}
+
+//-----------------------------------------------------------------------------------------------------------------------------
+// Function for ai game play \\
+
+function available() {
+    for (let cell of board) {
+        if (cell.textContent = "") {
+        }
```

Comment:
The first way to attempt creating an AI might be to think about how you could get it to just pick a **[random](https://www.w3schools.com/js/js_random.asp)** square that has not been selected. It might make things easier if you were to tell it to **push** the user's choice into an empty array, so it can be referenced later.